entrypoint.sh

```diff
@@ -190,6 +193,26 @@ function execute_push {
   local request="cloudsmith push ${options["action"]} ${options["format"]} $context ${options["file"]} $params $extra"
   echo $request
   eval $request
+
+  test -n "${options["tags"]}" && {
+    query="filename:${options["file"]}"
+    check_option_set "${options["version"]}" && {
+      query+=" version:${options["version"]}"
+    }
+
+    echo RUnninng tags branch
+    type cloudsmith
+    env
+    slug=$(python3 -c "import json, subprocess, sys
+context = sys.argv[1]
+query = sys.argv[2]
+response = subprocess.check_output(['cloudsmith', 'list', 'packages', context, '--output-format', 'pretty_json', '--query', query])
```

Comment:
Hmm, looks like `cloudsmith` is actually a bash function in tests so Python will not be able to run it.